README.md

```diff
@@ -30,6 +30,20 @@ npm install stripe --save
 yarn add stripe
 ```
 
+### Beta version
+
+Stripe has features in the beta phase that can be accessed via the beta version of this package.
+We would love for you to try these and share feedback with us before these features reach the stable phase.
+The beta versions can be installed in one of two ways 
+- To install the latest beta version, run the command `npm install stripe@beta --save`  
+- To install a specific beta version, replace the term "beta" in the above command with the version number like `npm install stripe@1.2.3-beta.1 --save` 
+
+> Note: There can be breaking changes between beta packages. Therefore we recommend pinning the beta version to a specific version in your package.json file so that you can install the same version each time without breaking changes unless you are intentionally looking for the latest beta package. 
```

Comment:
```suggestion
> Note: There can be breaking changes between beta versions. Therefore we recommend pinning the package version to a specific beta version in your package.json file so that you can install the same version each time without breaking changes unless you are intentionally looking for the latest beta version. 
```